### ISSUE TITLE
import/named does not detect a missing import

### ISSUE THREAD

**Issue by NickHeiner** [benmosher/eslint-plugin-import#2209]
`import/named` should report for the following, but does not:

```js
import { doesNotExist } from './upstream';
```

I've repro'd this in both `2.34.4` and `2.24.2`, and with both JS and TS: [repro repo](https://github.com/NickHeiner/repro/tree/master/cases). To repro this, run:

```
yarn workspace eslint-plugin-import-2209-v2.24.2 test
```

Possibly related:
* https://github.com/import-js/eslint-plugin-import/issues/2005

Did I do something wrong in my setup?

**Comment by soryy708** [comment]
Do you get any interesting logs/warnings if you set this environment variable?

```
DEBUG=eslint-plugin-import:*
```
